genome_browser/_genome_diagram.py

```diff
@@ -35,6 +34,9 @@ def xlimits(self):
 
         return min(flat), max(flat)
 
+    def add_target(self, target):
+        target and self.targets.append(target)
```

Comment:
Equivalent to:
```
if target:
    self.targets.append(target)
```
The one-liner form is sometimes ill-advised in multi-line methods - as it can obfuscate conditional indentation - but it makes for a nicer visual flow here.